users/src/main/java/io/crate/operation/auth/AuthenticationService.java

```diff
@@ -38,9 +43,27 @@
     private static final String KEY_METHOD = "method";
 
     private final Map<String, AuthenticationMethod> authMethodRegistry = new HashMap<>();
+
+    @VisibleForTesting
+    protected Map<String, Map<String, String>> hbaConf() {
+        return hbaConf;
+    }
+
     private Map<String, Map<String, String>> hbaConf;
```

Comment:
Can we add a comment that explains the structure, or rename the variable to make it clearer what the key and values represent?